### PR TITLE
New /identityProtection segment in beta

### PR DESCRIPTION
**Pull request by cloudhandler**

### CODE REVIEW

File: api-reference/beta/api/riskdetection-get.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ One of the following permissions is required to call this API. To learn more, in
 <!-- { "blockType": "ignored" } -->
 ```http
 GET /riskDetections/{id}
+GET /identityProtection/riskDetections/{id}
```
Comment (dkershaw10) [inline api-reference/beta/api/riskdetection-get.md:33]: You also need to update the path in the Examples sections...

Comment (cloudhandler) [reply api-reference/beta/api/riskdetection-get.md:33]: updated

File: api-reference/beta/api/riskdetection-get.md [hunk 2]
```diff
@@ -56,6 +57,7 @@ Here is an example of the request.
 }-->
 ```msgraph-interactive
 GET https://graph.microsoft.com/beta/riskDetections/c2b6c2b9-dddc-acd0-2b39-d519d803dbc3
+GET https://graph.microsoft.com/beta/identityProtection/riskDetections/c2b6c2b9-dddc-acd0-2b39-d519d803dbc3
```
Comment (Lauragra) [inline api-reference/beta/api/riskdetection-get.md:60]: Are you adding two requests in one example? Please make these separate examples.

**Comment by Lauragra** [comment]
@cloudhandler , FYI, I think your validation errors are related to the fact the request examples include more than one HTTP request. Please create a separate numbered example in each topic for the second request.

**Comment by cloudhandler** [comment]
> @cloudhandler , FYI, I think your validation errors are related to the fact the request examples include more than one HTTP request. Please create a separate numbered example in each topic for the second request.

Ah that makes sense, I'll update

File: api-reference/beta/api/riskdetection-get.md (hunk 2 above)
Comment (cloudhandler) [reply api-reference/beta/api/riskdetection-get.md]: Fixed

File: api-reference/beta/api/riskdetection-get.md [hunk 3]
```diff
@@ -110,7 +111,72 @@ Content-type: application/json
     "additionalInfo": "[{\"Key\":\"userAgent\",\"Value\":\"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.106 Safari/537.36\"}]"
 }
 ```
+## Example 2
```
Comment (Lauragra) [inline api-reference/beta/api/riskdetection-get.md:114]: We actually need a single H2, Examples, and H3s for Example 1 and Example 2, like you have in the -list topic. Make sure that the Request and Response headings are H4s (not H5s), and add the unique title to the H3 to clarify what each example does.

Comment (cloudhandler) [reply api-reference/beta/api/riskdetection-get.md]: Fixed the headings and added unique titles (except for list history item, as that's only ever for one user)

**Comment by cloudhandler** [comment]
@dkershaw10 - updated examples

**Comment by cloudhandler** [comment]
@dkershaw10 - now that changes have been made, can we publish?

File: api-reference/beta/api/riskdetection-get.md (hunk 3 above)
Comment (cloudhandler) [reply api-reference/beta/api/riskdetection-get.md]: @Lauragra - can we merge this or are additional changes needed?

**Pull request merged**